app/views/curation_concerns/single_use_links_viewer/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div >
+  <h1 class="lower"><%= @asset %></h1>
+    <h2 class="non lower">Actions</h2>
+    <p>
+      <%= link_to "Download (can only be used once)", @download_link %>
+    </p>
+  <h2> Descriptions:</h2>
+
+  <div class="well active" id="descriptions_display">
+    <table class="table table-striped">
+      <tbody>
+      <% present_terms(@presenter) do |r, term| %>
```

Comment:
Curation Concerns doesn't use the presenter to render the Generic File metadata. Lines 7 through 21 can be replaced with:
```
<%= render "curation_concerns/base/attributes", curation_concern: @asset %>
```